src/service-override/extensionGallery.ts

```diff
@@ -82,6 +85,8 @@ export default function getServiceOverride (options: ExtensionGalleryOptions = {
     [IRemoteExtensionsScannerService.toString()]: new SyncDescriptor(RemoteExtensionsScannerService, [], true),
     [IExtensionTipsService.toString()]: new SyncDescriptor(ExtensionTipsService, [], true),
     [IRemoteUserDataProfilesService.toString()]: new SyncDescriptor(RemoteUserDataProfilesService, [], true),
-    [IWorkbenchExtensionEnablementService.toString()]: new SyncDescriptor(ExtensionEnablementService, [], true)
+    [IWorkbenchExtensionEnablementService.toString()]: new SyncDescriptor(ExtensionEnablementService, [], true),
+    [IExtensionUrlHandler.toString()]: new SyncDescriptor(ExtensionUrlHandler, [], true),
+    [IAuthenticationService.toString()]: new SyncDescriptor(AuthenticationService, [], true)
```

Comment:
Shouldn't this one be in its own service override?

Reply:
Yeah, I can split this one out! I was thinking it would only make sense to need this with the extension marketplace, but on second thought I'm wrong on that.

Reply:
Moved!